front/src/applications/operationalStudies/components/SimulationResults/ChartHelpers/ChartHelpers.ts

```diff
@@ -303,7 +303,7 @@ export const interpolateOnTime = (
       if (listValue === 'speed' || listValue === 'speeds') {
         const comparator = dataSimulation?.[listValue] || dataSimulation?.[listValue][0];
         if (comparator) {
-          const timeBisect = d3.bisector<any, any>((d) => d['time']).left;
+          const timeBisect = d3.bisector<any, any>((d) => d.time).left;
```

Comment:
A such you are not certain to get a time of a left

Reply:
This modification has been made by eslint, I think the syntax is correct here